modules/core/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryResetCacheDetailMetricsTask.java

```diff
@@ -23,10 +23,10 @@
 import org.apache.ignite.internal.visor.VisorOneNodeTask;
 
 /**
- * Reset query detail metrics.
+ * Reset query detailed metrics from cache context only. For compatibility when node does not have indexing module.
```

Comment:
Lets change javadoc:
This task should be used to reset query metrics when indexing feature not available.